kolibri/plugins/coach/assets/src/views/reports/ReportsLessonListPage.vue

```diff
@@ -188,10 +187,10 @@
       },
     },
     $trs: {
-      // activeLessons: 'Active lessons',
-      // inactiveLessons: 'Inactive lessons',
-      // noActiveLessons: 'No active lessons',
-      // noInactiveLessons: 'No inactive lessons',
+      activeLessons: 'Active lessons',
+      inactiveLessons: 'Inactive lessons',
+      noActiveLessons: 'No active lessons',
+      noInactiveLessons: 'No inactive lessons',
```

Comment:
Delete the string, not needed.

Reply:
@radinamatic `printLabel` string seems to be used for file names during exports(Ln 184) + also as a report header(Ln 10)